src/services/profile/index.jsx

```
@@ -0,0 +1,41 @@
+export const ImageCard = ({ image, title, subTitle }) => {
```

ESLint

[error] 'image' is missing in props validation

(react/prop-types)